test/python_code/generate_py_data.jl

```diff
@@ -0,0 +1,93 @@
+# Setup python env
```

Comment:
we can remove this file, no?

Reply:
oh, yea, this whole directory can go now d9b7954